scholly/service/core/push.js

```diff
@@ -146,13 +152,13 @@ async function createPagesSite(ghUsername, ghToken, repoName) {
       data: pagesBody,
     });
     if (createPagesResponse.status === 201) {
-      console.log('Pages site successfully created');
+      logger.log('Pages site successfully created');
```

Comment:
nit: do you mean `logger.info`?

Reply:
hmmm interesting, I vividly remember fixing this. but anyways, thanks for catching it, I'll fix it now